### ISSUE TITLE
disable eslint-plugin-vue-i18n on a single component

### ISSUE THREAD

**Issue by miteyema** [kazupon/eslint-plugin-vue-i18n#22]
For one particular component, I don't want to activate i18n, so I did the following:
```
<script>
export default {
  nuxtI18n: false
}
</script>
```
But then eslint-plugin-vue-i18n is still complaining I have raw text in that component.
I have added `/* eslint-disable vue-i18n/no-raw-text */` inside the script tags, but it's not working and is still giving me warnings about the raw text
```
<script>
/* eslint-disable vue-i18n/no-raw-text */
export default {
  nuxtI18n: false
}
</script>
```

Is this a bug or a known issue?

**Comment by kazupon** [comment]
Thank you for your reporting!

hmm... That's is bug ...

**Comment by kazupon** [comment]
Hi
Unfortunately, I could not reproduce. 😞 
Could you provide minimum reproduction envirouments, please? 🙏

**Comment by miteyema** [comment]
Hi, thanks already for having a look at the problem.

Please see https://github.com/miteyema/nuxt-i18n-demo
- setup: nuxt + nuxt-i18n eslint-plugin-vue-i18n
- `privacy.vue` page with `nuxtI18n: false` and `/* eslint-disable vue-i18n/no-raw-text */`

However, when I do `npm run dev`, I still get this:

<img width="568" alt="Screenshot 2019-07-23 at 10 46 12" src="https://user-images.githubusercontent.com/2599590/61674501-75a40200-ad37-11e9-99b8-f2d5c3090d74.png">

**Comment by kazupon** [comment]
Thank you for your reproduction codes!

I've checked your repo.
In this case, you do not disable it in your javascript code, but you need to html comment the following:

```vue
<template>
  <div>
    <!-- eslint-disable vue-i18n/no-raw-text -->
    <h1 class="title">
      Privacy
    </h1>
    <!-- eslint-disable vue-i18n/no-raw-text -->
    <h2 class="subtitle">
      As I don't have time to translate, this will just stay in English.
      <nuxt-link :to="localePath('index')">
        Go back
      </nuxt-link>
    </h2>
  </div>
</template>
```

**Comment by miteyema** [comment]
That makes sense, thank you for helping me out!

FYI: I just put `<!-- eslint-disable vue-i18n/no-raw-text -->` once at the top of the page and that also seems to work, e.g.:
```vue
<!-- eslint-disable vue-i18n/no-raw-text -->
<template>
  <div>
    <h1 class="title">
      Privacy
    </h1>
    <h2 class="subtitle">
      As I don't have time to translate, this will just stay in English.
      <nuxt-link :to="localePath('index')">
        Go back
      </nuxt-link>
    </h2>
  </div>
</template>
```

**Comment by kazupon** [comment]
yeah it's okay too! 👍
